source/blog/2019-01-06-ovirt-openshift-part-1.html.md

```diff
@@ -0,0 +1,148 @@
+
+For the sake of simplicity, this installation will be an all-in-one OpenShift cluster, on a single VM.  
+On top of that, in the 2nd post, we will run a classic web stack, a Java application with a simple REST-API endpoint + Postgres. Postgres will get a persistent volume from oVirt using its flexvolume driver.
+
+
+
+<a style="background-color:black;color:white;text-decoration:none;padding:4px 6px;font-family:-apple-system, BlinkMacSystemFont, &quot;San Francisco&quot;, &quot;Helvetica Neue&quot;, Helvetica, Ubuntu, Roboto, Noto, &quot;Segoe UI&quot;, Arial, sans-serif;font-size:12px;font-weight:bold;line-height:1.2;display:inline-block;border-radius:3px" href="https://unsplash.com/@soroushgolpoor?utm_medium=referral&amp;utm_campaign=photographer-credit&amp;utm_content=creditBadge" target="_blank" rel="noopener noreferrer" title="Download free do whatever you want high-resolution photos from Soroush golpoor"><span style="display:inline-block;padding:2px 3px"><svg xmlns="http://www.w3.org/2000/svg" style="height:12px;width:auto;position:relative;vertical-align:middle;top:-2px;fill:white" viewBox="0 0 32 32"><title>unsplash-logo</title><path d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"></path></svg></span><span style="display:inline-block;padding:2px 3px">Soroush golpoor</span></a>
+
+
+
+<script id="asciicast-219956" src="https://asciinema.org/a/219956.js" async></script>
+
+## Single shell file installation
+
+Dropping to shell - this [install.sh](https://github.com/oVirt/ovirt-openshift-extensions/blob/master/automation/ci/install.sh) is a wrapper for installing  the ovirt-openshift-installer container, it uses ansible-playbook and has two main playbooks: install_okd.yaml and install_extensions.yaml. The latter is mainly for installing ovirt storage plugins.
+
+The install.sh script has one dependency, it needs to have 'podman' installed on the host, while all the rest runs inside a container.
+
+The only dependency (except from running oVirt datacenter) is podman:
+```console
+[bastion ~]# dnf install podman
+``` 
+
+[For other ways to install podman consult the readme](https://github.com/containers/libpod/blob/master/docs/tutorials/podman_tutorial.md)
+
+If you can't install `podman` docker will be fine as well, just edit the install.sh, and substitute podman for docker.
+
+### Get the install.sh and customize
+```console
+[bastion ~]# curl -O "https://raw.githubusercontent.com/oVirt/ovirt-openshift-extensions/master/automation/ci/{install.sh,customization.yaml}"
+```
+
+Edit the `customization.yaml`:
+
+- Put the engine details in engine_url
+  ```yaml
+  engine_url: https://ovirt-engine-fqdn/ovirt-engine/api
+  ```
+
+- Choose the oVirt cluster and data domain you want, if you don't want 'Default'
+  ```yaml
+  openshift_ovirt_cluster: yours
+  openshift_ovirt_data_store: yours
+   ```
+- Unmark the memory check if you don't change the VM default memory for this install is 8Gb
```

Comment:
This need rephrasing:
Unmark the memory check if you don't change the VM default memory for this install is 8Gb

Reply:
done